src/view/widgets/slider_widget.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+from bokeh.models.formatters import PrintfTickFormatter
+import panel as pn
+from view.widgets.widget import UIWidget
+
+from view import ui_constants
```

Comment:
"Kleinigkeit": In allen anderen widgets wurden die ui_constants as c importiert, würde ich einheitlich machen (welche Option wäre mir egal ;)